src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
you can remove flex and nothing will change

Reply:
It was my first commit, but not last